### PR TITLE
Update macOS Beta version System Requirements

### PR DESCRIPTION
**Pull request by rodrigocmoreira**
The idea of this PR is to make it clear what are the System Requirements to run the macOS Beta version.

### CODE REVIEW

File: content/en/docs/refguide/installation/system-requirements.md [hunk 1]
```diff
@@ -31,6 +35,10 @@ These are the known limitations for Mac:
 * If you have already installed JDK previously, it may not be picked up properly during installation. You can either configure this manually or remove all references to JDK and run the installer again.
 * The installer is currently not notarized yet; you bypass the dialog box by using right-click > **Open**.
 
+{{% alert color="warning" %}}
+Administrator rights are required to install Mendix Studio Pro Beta for macOS.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/installation/system-requirements.md]: This is how we announced this feature in the release notes (Studio Pro on Mac (Beta)).

File: content/en/docs/refguide/installation/system-requirements.md [hunk 2]
```diff
@@ -20,7 +20,11 @@ Mendix Studio Pro has the requirements listed in the following sections.
 
 [Mendix Studio Pro](/refguide/modeling/) 10 is supported on 64-bit versions of Windows 10 release 1809 and above; this also includes Windows 11. 
 
-Studio Pro 10.7 and above can run on an Apple Mac on ARM and Intel Mac running macOS Sonoma and above. Studio Pro versions 10.0 and above can run on the M series Apple Silicon Mac using [Parallels 17](https://www.parallels.com/) and Windows 11. Earlier versions of Studio Pro can run on Apple Mac [using Parallels](/refguide/using-mendix-studio-pro-on-a-mac/).
+Studio Pro 10.7 and above can run in a beta mode on an Apple Mac on ARM and Intel Mac running macOS Sonoma 14.0 and above. Studio Pro versions 10.0 and above can run on the M series Apple Silicon Mac using [Parallels 17](https://www.parallels.com/) and Windows 11. Earlier versions of Studio Pro can run on Apple Mac [using Parallels](/refguide/using-mendix-studio-pro-on-a-mac/).
+
+{{% alert color="info" %}}
+While Studio Pro on Mac is in [public beta](/releasenotes/beta-features/), Mendix can only verify support for the latest macOS version available. Support for earlier versions is not guaranteed and it is best to update macOS to the newest version when using Studio Pro.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/installation/system-requirements.md:26]: I rewrote this, could you please check that I understood what you meant correctly and did not change the meaning of it? @rodrigocmoreira

Comment (rodrigocmoreira) [reply content/en/docs/refguide/installation/system-requirements.md:26]: Hi @MariaShaposhnikova I normally see people mentioning run an application "in beta mode", not "in a beta mode". I would keep it like that. I think we can also change "on an Apple Mac on ARM and Intel Mac" to "on a ARM Mac and Intel Mac". Other than that it looks good to me.

File: content/en/docs/refguide/installation/system-requirements.md (hunk 1 above)
Comment (rodrigocmoreira) [reply content/en/docs/refguide/installation/system-requirements.md]: This is fine by me.

**Pull request merged**